corehq/form_processor/backends/couch/dbaccessors.py

```diff
@@ -126,6 +126,10 @@ def get_cases(case_ids, ordered=False):
         ]
 
     @staticmethod
+    def case_exists(case_id):
+        return CommCareCase.get_db().doc_exist(case_id)
```

Comment:
Yeah that seems fine to me. Looks like `get_case_with_lock` was also doing a naive `CommCareCase.get_db().get(case_id)` without checking its `case_type`, so whether or not this is super "correct" (which is a harder question), it does seem to be what we've always been doing.